main.tf

```diff
@@ -252,6 +252,12 @@ resource "aws_ec2_client_vpn_route" "default" {
   depends_on = [
     aws_ec2_client_vpn_network_association.default
   ]
+
+  timeouts {
+    create = "5m"
+    update = "5m"
```

Comment:
@rpadovani it looks like `update` is not a `timeouts` config option on this resource. Please remove this single line and I'll re-run tests + review. 

See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_client_vpn_route#timeouts for full info on the timeouts configuration for this resource.

Reply:
:facepalm: thanks for properly reviewing this, I added it without double-checking, I am sorry. Fixed.